resources/views/courses/index.blade.php

```diff
@@ -0,0 +1,271 @@
+                    <div class="fs-sm text-muted">
+                        <span>{{ $sections->count() }} sections</span> &bull; <span>{{ $course->chapters->count() }}
+                            lectures</span> &bull;
+                        <span>{{ $prettyDuration }} total
+                            length</span>
+                    </div>
+                </div>
+                <div>
+                    <div class="accordion" id="sectionAccordion">
+                        @foreach ($sections as $item)
+                            <div class="accordion-item">
+                                <h2 class="accordion-header" id="heading{{ $item->id }}">
+                                    <button class="px-2 accordion-button collapsed bg-neutral-50" type="button"
+                                        data-bs-toggle="collapse" data-bs-target="#collapse{{ $item->id }}"
+                                        aria-expanded="false" aria-controls="collapse{{ $item->id }}">
+                                        {{ $item->title }}
+                                    </button>
+                                </h2>
+                                <div id="collapse{{ $item->id }}" class="accordion-collapse collapse"
+                                    aria-labelledby="heading{{ $item->id }}" data-bs-parent="#sectionAccordion">
+                                    <div class="accordion-body ps-2">
+                                        @foreach ($chapters as $ch)
+                                            @if ($ch->section_id === $item->id)
+                                                <div
+                                                    class="py-2 d-flex justify-content-between ps-4 align-items-center">
+                                                    <div class="gap-2 d-flex align-items-center">
+                                                        @if ($ch->is_free)
+                                                            <x-lucide-play-circle class="w-4 h-4" />
+                                                        @else
+                                                            <x-lucide-lock class="w-4 h-4" />
+                                                        @endif
+                                                        <span>{{ $ch->title }}</span>
+                                                    </div>
+                                                    <div class="gap-2 d-flex align-items-center">
+                                                        @if ($ch->is_free)
+                                                            <span>
+                                                                <button
+                                                                    @click="previewChapter('{{ $ch->video_url }}', freeSections)"
+                                                                    class="btn">preview</button></span>
```

Comment:
highlight the preview button using underline.

<img width="811" alt="Screenshot 2023-11-15 at 12 53 48" src="https://github.com/ahmadrosid/learnwise/assets/29406408/970acf92-3ecc-4901-bdc1-282cce436e1d">